plugins/modules/postgresql_pg_hba.py

```diff
@@ -51,6 +51,10 @@
       - Type of the rule. If not set, C(postgresql_pg_hba) will only return contents.
     type: str
     choices: [ local, host, hostnossl, hostssl ]
+  comment:
+    description:
+      - A comment that will be placed in the same line behind the rule. See also parameter I(keep_comments_at_rules).
```

Comment:
```suggestion
      - A comment that will be placed in the same line behind the rule. See also the I(keep_comments_at_rules) parameter.
```

Reply:
BTW IIRC the term `option` is considered more canonical when we're talking about module parameters:)